### ISSUE TITLE
backward_G in half_gan_style.py

### ISSUE THREAD

**Issue by imransalam** [jessemelpolio/non-stationary_texture_syn#4]
Hi, this is more of a question, than an issue.
In the definition backward_G, we take gram_matrix of the real_B, but not of the targets. So to calculate loss we compare gram_matrix of source to simple (not gram_matrix) of targets. Any reason for that?

**Comment by jessemelpolio** [comment]
Hi, please refer to the definition of  GramMSELoss defined in vgg.py. 
`class GramMSELoss(nn.Module):
    def forward(self, input, target):
        out = nn.MSELoss()(GramMatrix()(input), target)
        return (out)
`
We actually calculate the gram matrix of fake image here and then calculate the MSE loss.

**Comment by imransalam** [comment]
Thank you for the clarification. I am going to close this now.
